src/Microsoft.VisualStudio.Web.CodeGeneration.Templating/TypeUtilities.cs

```diff
@@ -22,7 +21,6 @@ private static bool IsNonIntegerPrimitive(Type type)
                    || (type == typeof(double))
                    || (type == typeof(float))
                    || (type == typeof(Guid))
-                   || (type == typeof(string))
                    || (type == typeof(TimeSpan))
```

Comment:
Looks like I can't really remove this from here as this is going to break `CodeModelMetadata`.